Feature/register

HebaZakaria opened the pull request:
Register Task .

HebaZakaria committed:
Register Activity with Navigation

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+        android:textColorHint="@color/color_FFF7F1F4"
+        android:inputType="text"
+        android:padding="@dimen/name_padding"
+        android:textSize="13sp"
```
extract to dimens

ebnrdwan replied:
still exist

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+        android:textColorHint="@color/color_FFF7F1F4"
+        android:inputType="textEmailAddress"
+        android:padding="@dimen/email_padding"
+        android:textSize="13sp"
```
extract to dimens

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+        android:textColorHint="@color/color_FFF7F1F4"
+        android:inputType="numberPassword"
+        android:padding="@dimen/pass_padding"
+        android:textSize="13sp"
```
extract to dimens

ebnrdwan replied:
still exist , text size should be extracted

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+        android:textColorHint="@color/color_FDFBFB"
+        android:inputType="numberPassword"
+        android:padding="@dimen/confirm_pass_padding"
+        android:textSize="13sp"
```
extract to dimens

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+
+        />
+
+    <Button
```
button is not centered horizontally and very far from the checkbox "i agree" 
please review against the design again

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+
+            />
+
+        <TextView
```
string has typo and checkbox design not matching design

ebnrdwan replied:
this is optional for now but in real projects we have to match the design with 100% accuracy

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+        />
+
+    <EditText
+        android:id="@+id/confirm_password"
```
background underline should be more transparent white

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+
+        />
+
+    <EditText
```
validate against user 
username is not empty 
mail is correct 
password is more than 6 characters and match confirm password

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+        android:drawableLeft="@drawable/ic_pass"
+        android:hint="@string/password"
+        android:textColorHint="@color/color_FFF7F1F4"
+        android:inputType="numberPassword"
```
text type should be text password not just numbers

ebnrdwan commented on app/src/main/res/layout/fragment_register.xml:
```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
```
add back arrow on top left as the design

ebnrdwan reviewed (changes requested):
very good pull request, matching the architecture and writing a tidy code 
 you just need to test the output after finishing the development and take a sharp review against the design 
the out is what represents you to people to give him a lot of care and attention with love 

so gooooood so far, keep it up

HebaZakaria committed:
Register Activity with Navigation
 Validatig for Username,Password,E_mail and confirmPassword
 Extract for some dimentions
 Adding back arrow

HebaZakaria committed:
Register Activity with Navigation
 Validatig for Username,Password,E_mail and confirmPassword
 Extract for some dimentions
 Adding back arrow
 Some Editing for RelativeLayout and EditText

HebaZakaria committed:
Register Activity with Navigation
 Validatig for Username,Password,E_mail and confirmPassword
 Extract for some dimentions
 Adding back arrow
 Some Editing for RelativeLayout and EditText
 Adding the(inputType)for Password and ConfirmPassword

HebaZakaria committed:
Register Activity with Navigation
 Validatig for Username,Password,E_mail and confirmPassword
 Extract for some dimentions
 Adding back arrow
 Some Editing for RelativeLayout and EditText
 Adding the(inputType)for Password and ConfirmPassword
 Implement Register with Firebase Authentication
 Verification via E-mail

HebaZakaria committed:
Merge branch 'master' of https://github.com/ebnrdwan/Dokkan into feature/Register

# Conflicts:
#	app/build.gradle
#	app/src/main/java/com/engineering/dokkan/view/MainActivity.java
#	app/src/main/res/layout/activity_main.xml
#	app/src/main/res/values/colors.xml
#	app/src/main/res/values/dimens.xml
#	app/src/main/res/values/strings.xml
#	build.gradle

ebnrdwan committed:
fix profile
